### PR TITLE
[draft][no_ci] Add 2.2.0 release logs.

### PR DESCRIPTION
**Pull request by xwjiang2010**
Signed-off-by: xwjiang2010 <xwjiang2010@gmail.com>

## Why are these changes needed?

Add 2.2.0 release logs. 
This is a preview. Will collect final logs after all cps are done.

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by stephanie-wang — APPROVED** [review]

**Review by scv119 — APPROVED** [review]

**Review by scv119 — APPROVED** [review]

**Review by scv119 — APPROVED** [review]

File: release/release_logs/2.2.0/stress_tests/stress_test_many_tasks.json [hunk 1]
```diff
@@ -3,45 +3,45 @@
...
             "perf_metric_name": "stage_1_avg_iteration_time",
             "perf_metric_type": "LATENCY",
-            "perf_metric_value": 25.25241599082947
+            "perf_metric_value": 22.138524317741393
         },
         {
             "perf_metric_name": "stage_2_avg_iteration_time",
             "perf_metric_type": "LATENCY",
-            "perf_metric_value": 70.90980587005615
+            "perf_metric_value": 53.894669151306154
```
Comment (rickyyx) [inline release/release_logs/2.2.0/stress_tests/stress_test_many_tasks.json:16]: interesting - this is a bit surprising. We still hitting 70 in the master.

Comment (rickyyx) [reply release/release_logs/2.2.0/stress_tests/stress_test_many_tasks.json:16]: This was believed to be a grpc issue. Wonder if any dependency changes in the release branch?

Comment (xwjiang2010) [reply release/release_logs/2.2.0/stress_tests/stress_test_many_tasks.json:16]: This is a latency metric? So the smaller the better? 

I am not sure about dependency changes though. 

The following are all the dependencies
```
Local environment information | 1s
-- | --
  | [INFO 2022-12-04 20:48:29,442] glue.py: 218  Installed python packages:
  | adal==1.2.7
  | aiobotocore==2.4.1
  | aiohttp==3.8.3
  | aiohttp-cors==0.7.0
  | aiohttp-middlewares==2.2.0
  | aioitertools==0.11.0
  | aioredis==2.0.1
  | aiorwlock==1.3.0
  | aiosignal==1.3.1
  | anyio==3.6.2
  | anyscale==0.5.51
  | applicationinsights==0.11.10
  | argcomplete==1.12.3
  | async-timeout==4.0.2
  | asynctest==0.13.0
  | attrs==22.1.0
  | azure-cli-core==2.40.0
  | azure-cli-telemetry==1.0.8
  | azure-common==1.1.28
  | azure-core==1.26.1
  | azure-identity==1.10.0
  | azure-mgmt-compute==23.1.0
  | azure-mgmt-core==1.3.2
  | azure-mgmt-network==19.0.0
  | azure-mgmt-resource==20.0.0
  | backoff==1.10.0
  | backports.zoneinfo==0.2.1
  | bcrypt==4.0.1
  | blessed==1.19.1
  | boto3==1.24.59
  | botocore==1.27.59
  | brotlipy==0.7.0
  | cachetools==5.2.0
  | certifi==2022.9.24
  | cffi @ file:///opt/conda/conda-bld/cffi_1642701102775/work
  | charset-normalizer==2.1.1
  | click==8.1.3
  | cloudpickle==2.2.0
  | colorama==0.4.6
  | colorful==0.5.4
  | commonmark==0.9.1
  | conda==22.11.0
  | conda-content-trust @ file:///tmp/build/80754af9/conda-content-trust_1617045594566/work
  | conda-pack==0.6.0
  | conda-package-handling @ file:///tmp/build/80754af9/conda-package-handling_1649105789509/work
  | cryptography @ file:///croot/cryptography_1665612644927/work
  | cycler==0.11.0
  | Cython==0.29.26
  | decorator==5.1.1
  | Deprecated==1.2.13
  | distlib==0.3.6
  | dm-tree==0.1.7
  | docutils==0.19
  | expiringdict==1.2.2
  | fastapi==0.88.0
  | filelock==3.8.0
  | flatbuffers==22.11.23
  | flit_core @ file:///opt/conda/conda-bld/flit-core_1644941570762/work/source/flit_core
  | fonttools==4.38.0
  | freezegun==1.2.2
  | frozenlist==1.3.3
  | fsspec==2022.11.0
  | gitdb==4.0.10
  | GitPython==3.1.29
  | google-api-core==2.11.0
  | google-api-python-client==1.7.8
  | google-auth==2.15.0
  | google-auth-httplib2==0.1.0
  | google-cloud-secret-manager==2.12.6
  | google-oauth==1.0.1
  | googleapis-common-protos==1.57.0
  | gpustat==1.0.0
  | grpc-google-iam-v1==0.12.4
  | grpcio==1.51.1
  | grpcio-status==1.48.2
  | gym==0.23.1
  | gym-notices==0.0.8
  | h11==0.14.0
  | halo==0.0.31
  | httplib2==0.21.0
  | humanfriendly==10.0
  | humanize==4.4.0
  | idna==3.4
  | imageio==2.22.4
  | importlib-metadata==5.1.0
  | importlib-resources==5.10.0
  | isodate==0.6.1
  | Jinja2==3.1.2
  | jmespath==0.10.0
  | joblib==1.2.0
  | jsonpatch==1.32
  | jsonpointer==2.3
  | jsonschema==4.17.3
  | kiwisolver==1.4.4
  | knack==0.10.1
  | kubernetes==25.3.0
  | lightgbm==3.3.3
  | lightgbm-ray @ git+https://github.com/ray-project/lightgbm_ray.git@a128ef456ed69e161ab378475052836c128ae27f
  | log-symbols==0.0.14
  | lz4==4.0.2
  | MarkupSafe==2.1.1
  | matplotlib==3.5.3
  | msal==1.18.0b1
  | msal-extensions==1.0.0
  | msgpack==1.0.4
  | msrest==0.7.1
  | msrestazure==0.6.4
  | multidict==6.0.3
  | networkx==2.6.3
  | numpy==1.21.6
  | nvidia-ml-py==11.495.46
  | oauth2client==4.1.3
  | oauthlib==3.2.2
  | opencensus==0.11.0
  | opencensus-context==0.1.3
  | opentelemetry-api==1.1.0
  | opentelemetry-exporter-otlp==1.1.0
  | opentelemetry-exporter-otlp-proto-grpc==1.1.0
  | opentelemetry-proto==1.1.0
  | opentelemetry-sdk==1.1.0
  | opentelemetry-semantic-conventions==0.20b0
  | packaging==21.3
  | pandas==1.3.5
  | paramiko==2.12.0
  | pathspec==0.10.2
  | Pillow==9.3.0
  | pip==22.2.2
  | pkginfo==1.9.2
  | pkgutil_resolve_name==1.3.10
  | platformdirs==2.5.4
  | pluggy @ file:///tmp/build/80754af9/pluggy_1648042572264/work
  | portalocker==2.6.0
  | prometheus-client==0.13.1
  | proto-plus==1.22.1
  | protobuf==3.20.3
  | psutil==5.9.4
  | py==1.11.0
  | py-spy==0.3.14
  | pyarrow==7.0.0
  | pyasn1==0.4.8
  | pyasn1-modules==0.2.8
  | pycosat==0.6.3
  | pycparser @ file:///tmp/build/80754af9/pycparser_1636541352034/work
  | pydantic==1.9.2
  | PyGithub==1.57
  | Pygments==2.13.0
  | PyJWT==2.6.0
  | PyNaCl==1.5.0
  | pyOpenSSL @ file:///opt/conda/conda-bld/pyopenssl_1643788558760/work
  | pyparsing==3.0.9
  | pyrsistent==0.19.2
  | PySocks @ file:///tmp/build/80754af9/pysocks_1594394576006/work
  | python-dateutil==2.8.2
  | python-dotenv==0.21.0
  | pytz==2022.6
  | pytz-deprecation-shim==0.1.0.post0
  | PyWavelets==1.3.0
  | PyYAML==6.0
  | ray @ https://s3-us-west-2.amazonaws.com/ray-wheels/releases/2.2.0/d40c1ca59a88ed87f3011a68cd3abcf061279bed/ray-2.2.0rc0-cp37-cp37m-manylinux2014_x86_64.whl
  | -e git+https://github.com/ray-project/ray.git@d40c1ca59a88ed87f3011a68cd3abcf061279bed#egg=ray_release&subdirectory=release
  | redis==3.5.3
  | requests==2.28.1
  | requests-oauthlib==1.3.1
  | retry==0.9.2
  | rich==12.6.0
  | rsa==4.9
  | ruamel.yaml @ file:///tmp/build/80754af9/ruamel.yaml_1600552829606/work
  | ruamel.yaml.clib @ file:///opt/conda/conda-bld/ruamel.yaml.clib_1641552548372/work
  | s3transfer==0.6.0
  | scikit-image==0.19.3
  | scikit-learn==1.0.2
  | scipy==1.7.3
  | setproctitle==1.2.2
  | setuptools==65.5.0
  | shellingham==1.5.0
  | six==1.13.0
  | slackclient==2.9.4
  | smart-open==6.2.0
  | smmap==5.0.0
  | sniffio==1.3.0
  | spinners==0.0.24
  | starlette==0.22.0
  | tabulate==0.9.0
  | tensorboardX==2.5.1
  | termcolor==2.1.1
  | threadpoolctl==3.1.0
  | tifffile==2021.11.2
  | toml==0.10.2
  | toolz @ file:///croot/toolz_1667464077321/work
  | tqdm @ file:///opt/conda/conda-bld/tqdm_1647339053476/work
  | typer==0.7.0
  | typing_extensions==4.4.0
  | tzdata==2022.7
  | tzlocal==4.2
  | uritemplate==3.0.1
  | urllib3==1.26.13
  | uvicorn==0.20.0
  | virtualenv==20.17.0
  | wcwidth==0.2.5
  | websocket-client==1.4.2
  | wheel==0.37.1
  | wrapt==1.14.1
  | xgboost==1.6.2
  | xgboost-ray @ git+https://github.com/ray-project/xgboost_ray.git@ecca2c63385841a0a1938f5edc349893e5ac63fc
  | yarl==1.8.2
  | zipp==3.11.0


```

Comment (rickyyx) [reply release/release_logs/2.2.0/stress_tests/stress_test_many_tasks.json:16]: yeah - it's actually an unexpected improvement. 

I see,maybe grpcio 1.51 has some improvements? But if that's the case, we should be able to see that in master soon in the next run maybe.

Comment (xwjiang2010) [reply release/release_logs/2.2.0/stress_tests/stress_test_many_tasks.json:16]: grpcio 1.51 is out 11/29 last week. So if this is causing that, I would imagine master should already see it.

Actually this is the most recent run on master: https://buildkite.com/ray-project/release-tests-branch/builds/1241#0184e45c-fdf5-4766-bcc4-0b147ce517d7

It's measuring similar latency metrics as well! So I guess good for us!

**Pull request merged**